app/uk/gov/hmrc/uknwauthcheckerapi/models/IntegrationFrameworkError.scala

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.uknwauthcheckerapi.models
+
+sealed trait IntegrationFrameworkError
+
+object IntegrationFrameworkError {
+  case class InternalUnexpectedError(cause: Option[Throwable]) extends IntegrationFrameworkError
+  case class BadGateway(reason: String, code: Int) extends IntegrationFrameworkError
```

Comment:
This is not a supported statusf from EIS/PDS

Reply:
Removed stale classes.